src/status_im/hardwallet/core.cljs

```diff
@@ -1326,6 +1367,8 @@
                        (assoc-in [:hardwallet :wallet-address] wallet-address)
                        (assoc-in [:hardwallet :encryption-public-key] encryption-public-key)
                        (assoc-in [:hardwallet :keycard-instance-uid] (or instance-uid' instance-uid))
```

Comment:
this one is actually an `(update-in [:hardwallet :keycard-instance-uid] #(if % % instance-uid))`

Reply:
yes, i've changed it to `update`